models/articles-model.js

```diff
@@ -11,8 +11,37 @@ exports.selectArticlesById = (params) => {
   });
 };
 
-exports.selectArticles = () => {
-  //hard coded for now, intending to change if adding queries
+exports.selectArticles = (params, queries) => {
+  let { sort_by, order, topic } = queries;
+
+  switch (sort_by) {
+    case "article_id":
+    case "title":
+    case "topic":
+    case "author":
+    case "comment_count":
+    case "created_at":
+    case "votes":
+    case "article_img_url":
+      break;
+    case undefined:
+      sort_by = "created_at";
+      break;
+    default:
+      return Promise.reject({ code: 400, msg: "Bad Request", custom: true });
+  }
```

Comment:
A more concise way to solve this is by placing the valid sort-bys in an array, then using . includes to check if the sort_by that has been passed to the function is valid or not.

That would make your function more readable and easily understandable, and is a more expected way of handling this.